README.md

```diff
@@ -40,6 +40,23 @@ Optionally you can pass a list of services to be rebuilt:
 ./bootstrap.sh up api web
 ```
 
+## Services pre-run setup
+Some services will need to perform a certain set of actions before they are able to start successfully. This might include eg. running migrations, setting up fixtures, etc.
+
+To accomodate this, this script will run all `./setup/*.sh` scripts defined, every time they change ssince the last setup run (as per last git modification timestamp).
```

Comment:
As here we have `./setup/*.sh` I think we should either:
1) Add this folder to the repository
2) Rephrase the description of `Services pre-run setup` and say something like `You can add `setup` folder with your setup configuration blah blah...`